src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3023,6 +3023,8 @@ private static bool AreCloseEnough(Symbol original, Symbol updated)
             {
                 (LambdaSymbol l, NamedTypeSymbol n) _ when n.IsDelegateType() => AreLambdaAndNewDelegateSimilar(l, n),
                 (FieldSymbol { ContainingType: { IsTupleType: true }, TupleElementIndex: var oi } originalField, FieldSymbol { ContainingType: { IsTupleType: true }, TupleElementIndex: var ui } updatedField) =>
+                    // For tuples with their indexes being private fields, such as in `InferredName_ConditionalOperator_UseSite`, when we do reinference we don't end up inferring the resulting type correctly,
```

Comment:
>indexes [](start = 45, length = 7)

Aleksey commented:

> I am not sure what "indexes" is referring to. Could you please elaborate?

Reply:
Added a more elaborate comment.

---
In reply to: [551560779](https://github.com/dotnet/roslyn/pull/50111#discussion_r551560779) [](ancestors = 551560779)

Reply:
Actually, this comment can just removed now that `AsMember` is correctly handling tuple element indexes for error fields.

---
In reply to: [551560877](https://github.com/dotnet/roslyn/pull/50111#discussion_r551560877) [](ancestors = 551560877,551560779)